Code/EnginePlugins/OpenXRPlugin/init.cmake

```diff
@@ -2,7 +2,7 @@
 ### OpenXR support
 ######################################
 
-set (EZ_BUILD_OPENXR OFF CACHE BOOL "Whether support for OpenXR should be added")
+set (EZ_BUILD_OPENXR ON CACHE BOOL "Whether support for OpenXR should be added")
```

Comment:
Is this change intended?

Reply:
Yes, OpenXR plugin is built by default now (which includes local builds and CI) so that it doesn't deteriorate anymore.